audformat/core/database.py

```diff
@@ -208,6 +209,34 @@ def segments(self) -> pd.MultiIndex:
         assert isinstance(index, pd.MultiIndex)
         return index.drop_duplicates()
 
+    def contains_nonunique_path(
```

Comment:
To be in-line with `db.drop_file()`, `db.files`, should we maybe call it `contains_nonunique_file()`. And would it also make sense to return `True` if all files are `unique` instead? Then we could simplify to  `contains_unique_files()`.

Reply:
I would prefer to stay with the current behavior as it is faster (in almost all cases) to test if one file has a non-unique path than to check that all files have unique paths.

Reply:
But that's fine, just invert the return value :)

Reply:
Good point, I changed to `contains_unique_files()`